<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CuteClassClass</class>
 <widget class="QWidget" name="CuteClassClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1212</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CuteClass</string>
  </property>
  <widget class="QLabel" name="label_showProcessImage">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>90</y>
     <width>461</width>
     <height>461</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="YHradioButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>油画效果</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="QBradioButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>铅笔画效果</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="KTradioButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>卡通效果</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enterpushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Adobe Arabic&quot;;
text-decoration: underline;</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelpushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>330</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Adobe Arabic&quot;;
text-decoration: underline;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
